admin/bin/check-ps-report-for-corrupt-data.js

```diff
@@ -0,0 +1,380 @@
+#!/usr/bin/env node
+'use strict'
+const fs = require('fs')
+const csv = require('fast-csv')
+const startTime = Date.now()
+const R = require('ramda')
+const RA = require('ramda-adjunct')
+
+// const timeRegex =  /^\d{1,2}:\d{2}:\d{2} (AM|PM)$/i // Not quite right - PM instead pm and single digit hours
```

Comment:
delete comment if obsolete please

Reply:
I'd prefer to leave it as it details the currently passing regex.  Also, when run against a dataset of 400K lines, it reports multiple times per line.  That's when you use the commented out one, as a sanity saver. 

I would suggest our next steps are to confirm with the PS team if this is acceptable and then consolidate the spec, script and output to conform to the standard.